src/components/common/Header.js

```diff
@@ -1,8 +1,16 @@
 import React, {PropTypes} from 'react';
+import backArrow from '../../res/images/common/arrow-left.png';
+import { CONTENTS } from '../../enums/enums';
+
+export const Header = ({ style, title, withBackButton, onBackPressed }) => {
+
+  const back = () => {
+    onBackPressed(CONTENTS.Home);
+  };
 
-export const Header = ({ style, title }) => {
   return(
     <div className={style}>
+      {withBackButton && <a onClick={back}><img className="back-button" src={backArrow} /></a>}
```

Comment:
Empty tags should be self-closing. In this case:
`<a onClick={back}><img className="back-button" src={backArrow} />`

Just a minor comment but to get in the styling. Did you set up a linter? If not, let me know and I'll help you with that ;)